incubator/hnc/internal/validators/namespace_test.go

```diff
@@ -12,29 +12,53 @@ import (
 )
 
 func TestDeleteSubNamespace(t *testing.T) {
-	// Create a namespace with owner annotation.
+	// Create a namespace instance a.
 	sub := &corev1.Namespace{}
 	sub.Name = "a"
```

Comment:
Maybe move this into the `t.Run` portion of the test? That's where we actually use it, plus it will ensure we get a fresh copy for each subtest.